### PR TITLE
Outreach (destination) documentation

### PR DESCRIPTION
**Pull request by mohan-h**
Initial commit

### CODE REVIEW

File: help/destinations/catalog/crm/outreach.md [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+keywords: crm;CRM;crm destinations;Outreach;Outreach crm destination
+title: Outreach connection
+description: The Outreach destination allows you to export your account data and activate it within Outreach for your business needs.
+---
+
+# [!DNL Outreach] connection
+
+## Overview {#overview}
+
+[Outreach](https://www.outreach.io/) is a Sales Execution Platform with the most B2B buyer-seller interaction data in the world and significant investments in proprietary AI technologies to translate sales data into intelligence, Outreach helps revenue organizations automate sales engagement and act on revenue intelligence to improve their efficiency, predictability, and growth.
```
Comment (vfalon) [inline help/destinations/catalog/crm/outreach.md]: Line 11: Full stop after "translate sales data into intelligence".

File: help/destinations/catalog/crm/outreach.md [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+
+Outreach uses OAuth 2 with Authorization Grant as the authentication mechanism to communicate with the Outreach Update Resource API. Instructions to authenticate to your Outreach instance are further below, in the [Authenticate to destination](#authenticate) section.
+
+## Use cases {#use-cases}
+
+As a marketer, you can deliver personalized experiences to your prospects, based on attributes from their Adobe Experience Platform profiles. You can build segments from your offline data and send these segments to Outreach, to display in the prospects’ feeds as soon as segments and profiles are updated in Adobe Experience Platform.
+
+## Prerequisites {#prerequisites}
+
+### Prerequisites in Experience Platform {#prerequisites-in-experience-platform}
```
Comment (vfalon) [inline help/destinations/catalog/crm/outreach.md]: I would rephrase the headings as` Experience Platform prerequisites` and `Outreach prerequisites`.

File: help/destinations/catalog/crm/outreach.md [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+
+Before activating data to the Outreach destination, you must have a [schema](https://experienceleague.adobe.com/docs/experience-platform/xdm/schema/composition.html), a [dataset](https://experienceleague.adobe.com/docs/platform-learn/tutorials/data-ingestion/create-datasets-and-ingest-data.html?lang=en), and [segments](https://experienceleague.adobe.com/docs/platform-learn/tutorials/segments/create-segments.html?lang=en) created in [!DNL Experience Platform].
+
+### Prerequisites in Outreach {#prerequisites-destination}
+
+Note the following prerequisites in Outreach, in order to export data from Platform to your Outreach account:
+
+#### You need to have an Outreach account {#prerequisites-account}
+
+Go to the Outreach [sign in](https://accounts.outreach.io/users/sign_in) page to register and create an account, if you do not have one already. Also see support [page](https://support.outreach.io/hc/en-us/articles/207238607-Claim-Your-Outreach-Account).
```
Comment (vfalon) [inline help/destinations/catalog/crm/outreach.md]: See the Outreach [support page](https://support.outreach.io/hc/en-us/articles/207238607-Claim-Your-Outreach-Account) for more details.

File: help/destinations/catalog/crm/outreach.md [hunk 4]
```diff
@@ -0,0 +1,208 @@
...
+
+Refer to Adobe's documentation for [Segment Membership Details schema field group](https://experienceleague.adobe.com/docs/experience-platform/xdm/field-groups/profile/segmentation.html?lang=en) if you need guidance on segment statuses.
```
Comment (vfalon) [inline help/destinations/catalog/crm/outreach.md]: Line 35: why is this mentioned here? Shouldn't this be mentioned in the Experience Platform prerequisites section?

Also, you are linking to a page within the AEP docs repo, so please use a relative link to the Markdown file instead of the public Experience League page.

File: help/destinations/catalog/crm/outreach.md [hunk 5]
```diff
@@ -0,0 +1,208 @@
...
+Note down the items below before you authenticate to the Outreach CRM destination:
+
+| Credential | Description |
+| --- | --- | --- |
+| Email | Your Outreach account email |
+| Password | Your Outreach account password |
+
+#### Setup custom field labels
+
+Outreach supports custom fields for [prospects](https://support.outreach.io/hc/en-us/articles/360001557554-Outreach-Prospect-Profile-Overview). Refer to [How To add a custom field in Outreach](https://support.outreach.io/hc/en-us/articles/219124908-How-To-Add-a-Custom-Field-in-Outreach) for additional guidance. For ease of identification it is recommended to manually update the label's to their corresponding segment names instead of keeping the defaults. For example as below:
```
Comment (vfalon) [inline help/destinations/catalog/crm/outreach.md]: Line 48: labels instead of label's.

File: help/destinations/catalog/crm/outreach.md [hunk 6]
```diff
@@ -0,0 +1,208 @@
...
+
+Outreach settings page for prospects displaying custom fields.
```
Comment (vfalon) [inline help/destinations/catalog/crm/outreach.md]: Please enclose Outreach in [!DNL ] tags throughout the doc.

File: help/destinations/catalog/crm/outreach.md [hunk 7]
```diff
@@ -0,0 +1,208 @@
...
+![Outreach custom fields defaults](../../assets/catalog/crm/outreach/outreach-custom-fields.png)
+
+Outreach settings page for prospects displaying custom fields with *user-friendly* labels matching to the segment names. You can view the segment status on the prospect page against these labels.
+![Outreach custom fields with labels](../../assets/catalog/crm/outreach/outreach-custom-field-labels.png)
+
+> [!NOTE]
+>
+> When updating prospects the label names are not used they are for ease of identification only.
```
Comment (vfalon) [inline help/destinations/catalog/crm/outreach.md]: Please rephrase this note. Something like "Label names are not used when updating prospects. They are only used for ease of identification.".

File: help/destinations/catalog/crm/outreach.md [hunk 8]
```diff
@@ -0,0 +1,208 @@
...
+
+## Supported identities {#supported-identities}
+
+Outreach supports update of identities described in the table below. Learn more about [identities](/help/identity-service/namespaces.md).
+
+|Target Identity|Description|Considerations|
+|---|---|---|
+| OutreachId |Custom Outreach identifier that supports mapping of any identity.|Mandatory. You can send any [identity](../../../identity-service/namespaces.md) to the [!DNL Outreach] destination, as long as you map it to the `OutreachId`. |
```
Comment (vfalon) [inline help/destinations/catalog/crm/outreach.md]: Is OutreachId the name of the variable? If so, please enclose it in code tags. If it is shown as Outreach ID in the UI, please format it accordingly.

File: help/destinations/catalog/crm/outreach.md [hunk 9]
```diff
@@ -0,0 +1,208 @@
...
+
+## Connect to the destination {#connect}
+
+> [!IMPORTANT]
+> 
+> To connect to the destination, you need the **[!UICONTROL Manage Destinations]** [access control permission](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+
+To connect to this destination, follow the steps described in the [destination configuration tutorial](../../ui/connect-destination.md). In the configure destination workflow, fill in the fields listed in the two sections below.
+
+![Catalog](../../assets/catalog/crm/outreach/catalog.png)
```
Comment (vfalon) [inline help/destinations/catalog/crm/outreach.md]: This screen has five highlights on it, but users might not understand the order those are shown in. Maybe add some numbering next to them (1. Destinations, 2.Catalog, etc.).

File: help/destinations/catalog/crm/outreach.md [hunk 10]
```diff
@@ -0,0 +1,208 @@
...
+![Sample screenshot showing how to authenticate to Outreach](../../assets/catalog/crm/outreach/authenticate-destination-login-email.png)
+
+Next provide your password.
+
+![Sample screenshot showing how to authenticate to Outreach](../../assets/catalog/crm/outreach/authenticate-destination-login-password.png)
+
+*  **[!UICONTROL Username]**: Your Outreach account email.
+*  **[!UICONTROL Password]**: Your Outreach account password.
+
+If the details provided are valid, the UI displays a **Connected** status with a green check mark, you can then proceed to the next step.
```
Comment (vfalon) [inline help/destinations/catalog/crm/outreach.md]: Full stop after "green check mark".

File: help/destinations/catalog/crm/outreach.md [hunk 11]
```diff
@@ -0,0 +1,208 @@
...
+
+Read [Activate profiles and segments to streaming segment export destinations](../../ui/activate/activate-segment-streaming-destinations.md) for instructions on activating audience segments to this destination.
+
+### Mapping considerations and example {#mapping-considerations-example}
+
+To correctly send your audience data from Adobe Experience Platform to the Outreach destination, you need to go through the field mapping step. Mapping consists of creating a link between your Experience Data Model (XDM) schema fields in your Platform account and their corresponding equivalents from the target destination. To correctly map your XDM fields to the Outreach destination fields, follow these steps:
+
+1. In the Mapping step, click **[!UICONTROL Add new mapping]**, you will see a new mapping row on the screen.
```
Comment (vfalon) [inline help/destinations/catalog/crm/outreach.md]: Line 134: "In the [UICONTROL Mapping] step".

File: help/destinations/catalog/crm/outreach.md [hunk 12]
```diff
@@ -0,0 +1,208 @@
...
+![Add new mapping](../../assets/catalog/crm/outreach/add-new-mapping.png)
+
+1. In the select source field window, when selecting the source field choose the **[!UICONTROL Select identity namespace]** category and add the mappings desired.
```
Comment (vfalon) [inline help/destinations/catalog/crm/outreach.md]: Line 137: "[!UICONTROL Select source field] window"

File: help/destinations/catalog/crm/outreach.md [hunk 13]
```diff
@@ -0,0 +1,208 @@
...
+![Source mapping](../../assets/catalog/crm/outreach/source-mapping.png)
+
+1. In the select target field window, select the target field and choose the **[!UICONTROL Select identity namespace]** category and add the mappings desired.
```
Comment (vfalon) [inline help/destinations/catalog/crm/outreach.md]: Steps 3&4 should be one single step with 2 bullet points, one showing the namespace mapping, one showing the custom attributes mapping. Please reorganize this step around this structure. Something like this:

In the [!UICONTROL Select target field] screen, select the target field type:
* **[!UICONTROL Select identity namespace]**: select this option to map your source field to an identity namespace from the list.
* **[!UICONTROL Select custom attributes]**: select this option to map your source field to a custom attribute that you define in the [!UICONTROL Attribute name] field.

File: help/destinations/catalog/crm/outreach.md [hunk 14]
```diff
@@ -0,0 +1,208 @@
...
+![Target mapping using contactId](../../assets/catalog/crm/outreach/target-mapping.png)
+
+1. For custom attributes, in the select target field window, select the target field and choose the **[!UICONTROL Select custom attributes]** category, Next provide the desired target attribute name and add the mappings desired.
+![Target mapping using LastName](../../assets/catalog/crm/outreach/target-mapping-lastname.png)
+
+1. For instance, you could add the following mapping between your XDM profile schema and your [!DNL Outreach] instance:
```
Comment (vfalon) [inline help/destinations/catalog/crm/outreach.md]: Line 146: This step is an example that should be included in the previous step.

File: help/destinations/catalog/crm/outreach.md [hunk 15]
```diff
@@ -0,0 +1,208 @@
...
+
+    ||XDM Profile Schema|[!DNL Outreach] Instance| Mandatory|
```
Comment (vfalon) [inline help/destinations/catalog/crm/outreach.md]: What should the user understand from the Mandatory column? Is the Outreach ID identity mandatory mapping for everyone? If it is, we should call this out in the mapping step.  Also, the Attributes example doesn't have any mandatory column value. Should it be Yes/No?

File: help/destinations/catalog/crm/outreach.md [hunk 16]
```diff
@@ -0,0 +1,208 @@
...
+1. An example using these mappings is shown below:
+![Target mapping](../../assets/catalog/crm/outreach/mappings.png)
+
+Refer to [Outreach prospect documentation](https://api.outreach.io/api/v2/docs#prospect) for a comprehensive list of supported attributes.
+
+### Schedule segment export and example {#schedule-segment-export-example}
+
+When performing the [Schedule segment export](../../ui/activate/activate-segment-streaming-destinations.html?lang=en#scheduling) step you must manually map Platform segments to the custom field attribute in Outreach.
+
+To do this, select each segment, then enter the corresponding numeric value which corresponds to the *custom`N`* attribute from Outreach in the **[!UICONTROL Mapping ID]** field.
```
Comment (vfalon) [inline help/destinations/catalog/crm/outreach.md]: Let's spell out an example of this mandatory mapping ID. When reading this section, my understanding was that Mapping ID should have a value of custom1/custom2/custom3/etc. It doesn't look like that is the case from the screenshot.

**Review by vfalon — CHANGES_REQUESTED** [review]
Thanks for opening the PR. There are a few changes that should be addressed before we can approve it. See my comments in the review.

File: help/destinations/catalog/crm/outreach.md (hunk 8 above)
Comment (vfalon) [inline help/destinations/catalog/crm/outreach.md]: Let's add an example of an Outreach ID. What format does it have? What kind of mapping does it support? What can users map to an OutreachID?

File: help/destinations/catalog/crm/outreach.md (hunk 16 above)
Comment (vfalon) [inline help/destinations/catalog/crm/outreach.md]: Let's add a separate section explaining how to obtain this mapping ID, step by step, with screenshots from the Outreach UI.

File: help/destinations/catalog/crm/outreach.md (hunk 8 above)
Comment (mohan-h) [reply help/destinations/catalog/crm/outreach.md]: It is an id within Outreach, however to make it easier for the end user within AEP its called OutreachId.

File: help/destinations/catalog/crm/outreach.md (hunk 15 above)
Comment (mohan-h) [reply help/destinations/catalog/crm/outreach.md]: _What should the user understand from the Mandatory column? Is the Outreach ID identity mandatory mapping for everyone? If it is, we should call this out in the mapping step._ 
Added additional notes as suggested,

_Also, the Attributes example doesn't have any mandatory column value. Should it be Yes/No?_
It depends on the attributes that the end user wants to update. Hence there is no input provided. Have reworded, trust it would work.

File: help/destinations/catalog/crm/outreach.md (hunk 16 above)
Comment (mohan-h) [reply help/destinations/catalog/crm/outreach.md]: Thanks for highlighting have reworded.

Comment (mohan-h) [reply help/destinations/catalog/crm/outreach.md]: Have updated the content, the rewording should hopefully fix this explanation discrepancy. 
P.s. We already have screengrab's in pre-requisites explaining creation of custom fields.

File: help/destinations/catalog/crm/outreach.md (hunk 8 above)
Comment (mohan-h) [reply help/destinations/catalog/crm/outreach.md]: Thanks, have reworded.

File: help/destinations/catalog/crm/outreach.md [hunk 17]
```diff
@@ -0,0 +1,217 @@
...
+
+## Connect to the destination {#connect}
+
+> [!IMPORTANT]
+> 
+> To connect to the destination, you need the **[!UICONTROL Manage Destinations]** [access control permission](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+
+To connect to this destination, follow the steps described in the [destination configuration tutorial](../../ui/connect-destination.md). In the configure destination workflow, fill in the fields listed in the two sections below.
+
+![Catalog](../../assets/catalog/crm/outreach/catalog.png)
```
Comment (vfalon) [inline help/destinations/catalog/crm/outreach.md]: This screenshot looks pretty cluttered. I would remove it completely, since this step is standard across all destinations and is described in the article you already linked to.

File: help/destinations/catalog/crm/outreach.md [hunk 18]
```diff
@@ -0,0 +1,217 @@
...
+Read [Activate profiles and segments to streaming segment export destinations](../../ui/activate/activate-segment-streaming-destinations.md) for instructions on activating audience segments to this destination.
+
+### Mapping considerations and example {#mapping-considerations-example}
+
+To correctly send your audience data from Adobe Experience Platform to the [!DNL Outreach] destination, you need to go through the field mapping step. Mapping consists of creating a link between your Experience Data Model (XDM) schema fields in your Platform account and their corresponding equivalents from the target destination. To correctly map your XDM fields to the [!DNL Outreach] destination fields, follow these steps:
+
+1. In the "[!UICONTROL Mapping]" step, click **[!UICONTROL Add new mapping]**, you will see a new mapping row on the screen.
+![Add new mapping](../../assets/catalog/crm/outreach/add-new-mapping.png)
+
+1. In the "[!UICONTROL Select source field]" window, when selecting the source field choose the **[!UICONTROL Select identity namespace]** category and add the mappings desired.
```
Comment (vfalon) [inline help/destinations/catalog/crm/outreach.md]: Line 137: "[...] and add the desired mappings.".

File: help/destinations/catalog/crm/outreach.md [hunk 19]
```diff
@@ -0,0 +1,217 @@
...
+![Source mapping](../../assets/catalog/crm/outreach/source-mapping.png)
+
+1. In the "[!UICONTROL Select target field]" window, select the target field
```
Comment (vfalon) [inline help/destinations/catalog/crm/outreach.md]: Line 140: In the "[!UICONTROL Select target field]" window, select the type of target field that you want to map the Outreach ID to.

**Review by vfalon — CHANGES_REQUESTED** [review]
Documentation looks good. Just a few minor comments.

File: help/destinations/catalog/crm/outreach.md (hunk 17 above)
Comment (mohan-h) [reply help/destinations/catalog/crm/outreach.md]: If we remove it the user might not specifically choose the connector under the CRM category. Have updated with fewer highlights and no numbering (which is why I had avoided it in the first place).

File: help/destinations/catalog/crm/outreach.md (hunk 19 above)
Comment (mohan-h) [reply help/destinations/catalog/crm/outreach.md]: Have done the change but would read odd. Suggest to not explicitly mention Outreach Id as subsequent bullets cover not just the Identity namespace mapping but also other custom attributes for different fields.

File: help/destinations/catalog/crm/outreach.md [hunk 20]
```diff
@@ -0,0 +1,217 @@
...
+
+To correctly send your audience data from Adobe Experience Platform to the [!DNL Outreach] destination, you need to go through the field mapping step. Mapping consists of creating a link between your Experience Data Model (XDM) schema fields in your Platform account and their corresponding equivalents from the target destination. To correctly map your XDM fields to the [!DNL Outreach] destination fields, follow these steps:
+
+1. In the "[!UICONTROL Mapping]" step, click **[!UICONTROL Add new mapping]**, you will see a new mapping row on the screen.
+![Add new mapping](../../assets/catalog/crm/outreach/add-new-mapping.png)
+
+1. In the "[!UICONTROL Select source field]" window, when selecting the source field choose the **[!UICONTROL Select identity namespace]** category and add the desired mappings.
+![Source mapping](../../assets/catalog/crm/outreach/source-mapping.png)
+
+1. In the "[!UICONTROL Select target field]" window, select the type of target field that you want to map the Outreach ID to.
```
Comment (vfalon) [inline help/destinations/catalog/crm/outreach.md]: Line 140: Enclose Outreach ID in DNL tags.

File: help/destinations/catalog/crm/outreach.md [hunk 21]
```diff
@@ -0,0 +1,217 @@
...
+
+## Use cases {#use-cases}
+
+As a marketer, you can deliver personalized experiences to your prospects, based on attributes from their Adobe Experience Platform profiles. You can build segments from your offline data and send these segments to [!DNL Outreach], to display in the prospects’ feeds as soon as segments and profiles are updated in Adobe Experience Platform.
+
+## Prerequisites {#prerequisites}
+
+### Experience Platform Prerequisites {#prerequisites-in-experience-platform}
+
+Before activating data to the [!DNL Outreach] destination, you must have a [schema](/help/xdm/schema/composition.html), a [dataset](https://experienceleague.adobe.com/docs/platform-learn/tutorials/data-ingestion/create-datasets-and-ingest-data.html?lang=en), and [segments](https://experienceleague.adobe.com/docs/platform-learn/tutorials/segments/create-segments.html?lang=en) created in [!DNL Experience Platform].
```
Comment (vfalon) [inline help/destinations/catalog/crm/outreach.md]: Please run a search for "html" in this page and replace all HTML links with relative links to the the Markdown file in the repo, where needed. There are still instances of html pages.

**Review by vfalon — CHANGES_REQUESTED** [review]

File: help/destinations/catalog/crm/outreach.md (hunk 17 above)
Comment (mohan-h) [reply help/destinations/catalog/crm/outreach.md]: Have removed the screenshot.

File: help/destinations/catalog/crm/outreach.md [hunk 22]
```diff
@@ -0,0 +1,219 @@
...
```
Comment (vfalon) [inline help/destinations/catalog/crm/outreach.md]: Line 25: schema URL is linking to HTML. Please change to markdown file.

File: help/destinations/catalog/crm/outreach.md [hunk 23]
```diff
@@ -0,0 +1,219 @@
...
+
+#### Setup custom field labels
+
+[!DNL Outreach] supports custom fields for [prospects](https://support.outreach.io/hc/en-us/articles/360001557554-Outreach-Prospect-Profile-Overview). Refer to [How To add a custom field in Outreach](https://support.outreach.io/hc/en-us/articles/219124908-How-To-Add-a-Custom-Field-in-Outreach) for additional guidance. For ease of identification it is recommended to manually update the labels to their corresponding segment names instead of keeping the defaults. For example as below:
+
+[!DNL Outreach] settings page for prospects displaying custom fields.
+![[!DNL Outreach] custom fields defaults](../../assets/catalog/crm/outreach/outreach-custom-fields.png)
+
+[!DNL Outreach] settings page for prospects displaying custom fields with *user-friendly* labels matching to the segment names. You can view the segment status on the prospect page against these labels.
+![[!DNL Outreach] custom fields with labels](../../assets/catalog/crm/outreach/outreach-custom-field-labels.png)
```
Comment (vfalon) [inline help/destinations/catalog/crm/outreach.md]: One thing that applies to all destinations. To improve readability and accessibility for all our docs, we have been asked to use clearer alt-text for images. Please review all the alt-text in the page and update it to a form similar to this:

"Outreach UI screenshot showing custom fields with associated labels."

Use clear sentences that describe as accurate as possible what is shown in the image. This is especially useful for users who are unable to see the image and/or use screen readers.

File: help/destinations/catalog/crm/outreach.md [hunk 24]
```diff
@@ -0,0 +1,219 @@
...
+To fix this error, verify that the Mapping ID you provided in [!DNL Outreach] for your Platform segment is valid and exists within [!DNL Outreach].
+## Additional resources {#additional-resources}
+
+The [[!DNL Outreach] documentation](https://api.outreach.io/api/v2/docs/) has details on [Error Responses](https://api.outreach.io/api/v2/docs#error-responses) which you can use to debug any issues.
+
+### Limits
+
+The [!DNL Outreach] API is rate-limited on a per-user basis, with a fixed limit of 10,000 requests per one-hour period. If you encounter this limit you will encounter a `429` response with a message similar to *`You have exceeded your permitted rate limit of 10,000; please try again at 2017-01-01T00:00:00.`*
+
+Refer to the [[!DNL Outreach] documentation](https://api.outreach.io/api/v2/docs#:~:text=%3A00%22%20%7D%5D%20%7D-,Rate%20Limiting,requests%20per%20one%2Dhour%20period1) for additional details.
```
Comment (vfalon) [inline help/destinations/catalog/crm/outreach.md]: Line 219: That URL does not look right. I'm guessing it should just be a link to the home page.

**Review by vfalon — CHANGES_REQUESTED** [review]
Left a couple more comments.

File: help/destinations/catalog/crm/outreach.md [hunk 25]
```diff
@@ -0,0 +1,219 @@
...
+|---|---|---|
+| `OutreachId` | <ul><li>[!DNL Outreach] identifier, this is a numeric value corresponding to the prospect profile.</li><li>The ID must match the ID within the [!DNL Outreach] URL for the prospect being updated.</li><li>Refer to the [[!DNL Outreach] documentation](https://api.outreach.io/api/v2/docs#update-an-existing-resource) for further details.</li></ul>|Mandatory |
+
+## Export type and frequency {#export-type-frequency}
+
+Refer to the table below for information about the destination export type and frequency.
+
+| Item | Type | Notes |
+---------|----------|---------|
+| Export type | **[!UICONTROL Profile-based]** | <ul><li>You are exporting all members of a segment, together with the desired schema fields *(for example: email address, phone number, last name)*, according to your field mapping.</li><li> Platform segment statuses are exported to [!DNL Outreach] by specifying their corresponding custom field attribute in [!DNL Outreach], within the **[!UICONTROL Activate Destination]** > **[!UICONTROL Schedule segment export]** > **[!UICONTROL Mapping ID]** field.</li></ul> |
```
Comment (vfalon) [inline help/destinations/catalog/crm/outreach.md]: On line 74 we need to rephrase the second bullet point:

- Platform segment statuses are exported to [!DNL Outreach] by entering their [!DNL Outreach] custom field attribute in the [!UICONTROL Mapping ID] field, during the [!UICONTROL Schedule segment export] step.

**Review by vfalon — CHANGES_REQUESTED** [review]

File: help/destinations/catalog/crm/outreach.md (hunk 25 above)
Comment (mohan-h) [reply help/destinations/catalog/crm/outreach.md]: There is a section #schedule-segment-export-example should I also link to it ?

File: help/destinations/catalog/crm/outreach.md [hunk 26]
```diff
@@ -0,0 +1,219 @@
...
+Read [Activate profiles and segments to streaming segment export destinations](../../ui/activate/activate-segment-streaming-destinations.md) for instructions on activating audience segments to this destination.
+
+### Mapping considerations and example {#mapping-considerations-example}
+
+To correctly send your audience data from Adobe Experience Platform to the [!DNL Outreach] destination, you need to go through the field mapping step. Mapping consists of creating a link between your Experience Data Model (XDM) schema fields in your Platform account and their corresponding equivalents from the target destination. To correctly map your XDM fields to the [!DNL Outreach] destination fields, follow these steps:
+
+1. In the [!UICONTROL Mapping] step, click **[!UICONTROL Add new mapping]**, you will see a new mapping row on the screen.
+![Platform UI screenshot showing how to Add new mapping](../../assets/catalog/crm/outreach/add-new-mapping.png)
+
+1. In the [!UICONTROL Select source field] window, when selecting the source field choose the **[!UICONTROL Select identity namespace]** category and add the desired mappings.
```
Comment (vfalon) [inline help/destinations/catalog/crm/outreach.md]: Line 137: remove the "when selecting the source field" part. It's redundant.

File: help/destinations/catalog/crm/outreach.md [hunk 27]
```diff
@@ -0,0 +1,219 @@
...
+    * **[!UICONTROL Select custom attributes]**: select this option to map your source field to a custom attribute that you define in the [!UICONTROL Attribute name] field. Refer to [[!DNL Outreach] prospect documentation](https://api.outreach.io/api/v2/docs#prospect) for a comprehensive list of supported attributes.
+    ![Platform UI screenshot showing Target mapping using LastName.](../../assets/catalog/crm/outreach/target-mapping-lastname.png)
+
+    * For instance, depending on the values you want to update, add the following mapping between your XDM profile schema and your [!DNL Outreach] instance:
+        |XDM Profile Schema|[!DNL Outreach] Instance|
+        |---|---|
+        |`person.name.firstName`|`firstName`|
+        |`person.name.lastName`|`lastName`|
+
+1. An example using these mappings is shown below:
```
Comment (vfalon) [inline help/destinations/catalog/crm/outreach.md]: Line 158: this is not a step, it's an example. Please either indent it so it falls under the example above, or remove the numbering.

File: help/destinations/catalog/crm/outreach.md (hunk 25 above)
Comment (vfalon) [reply help/destinations/catalog/crm/outreach.md]: Yes, please link to that section, but also rephrase the bullet point.

File: help/destinations/catalog/crm/outreach.md [hunk 28]
```diff
@@ -0,0 +1,219 @@
...
+When checking a dataflow run, if you obtain the following error message: `Bad request reported while pushing events to the destination. Please contact the administrator and try again.`
+
+![Platform UI screenshot showing the Bad request Error.](../../assets/catalog/crm/outreach/error.png)
+
+To fix this error, verify that the Mapping ID you provided in [!DNL Outreach] for your Platform segment is valid and exists within [!DNL Outreach].
+## Additional resources {#additional-resources}
+
+The [[!DNL Outreach] documentation](https://api.outreach.io/api/v2/docs/) has details on [Error Responses](https://api.outreach.io/api/v2/docs#error-responses) which you can use to debug any issues.
+
+### Limits
```
Comment (vfalon) [inline help/destinations/catalog/crm/outreach.md]: The limits section is not an "Additional resource". Please move this after the prerequisites section, as a `Guardrails` H2 heading.

File: help/destinations/catalog/crm/outreach.md [hunk 29]
```diff
@@ -0,0 +1,219 @@
...
+
+[!DNL Outreach] uses OAuth 2 with Authorization Grant as the authentication mechanism to communicate with the [!DNL Outreach] Update Resource API. Instructions to authenticate to your [!DNL Outreach] instance are further below, within [Authenticate to destination](#authenticate) section.
+
+## Use cases {#use-cases}
+
+As a marketer, you can deliver personalized experiences to your prospects, based on attributes from their Adobe Experience Platform profiles. You can build segments from your offline data and send these segments to [!DNL Outreach], to display in the prospects’ feeds as soon as segments and profiles are updated in Adobe Experience Platform.
+
+## Prerequisites {#prerequisites}
+
+### Experience Platform Prerequisites {#prerequisites-in-experience-platform}
```
Comment (vfalon) [inline help/destinations/catalog/crm/outreach.md]: Sentence case for headings, please.

File: help/destinations/catalog/crm/outreach.md [hunk 30]
```diff
@@ -0,0 +1,219 @@
...
+
+Before activating data to the [!DNL Outreach] destination, you must have a [schema](/help/xdm/schema/composition.md), a [dataset](https://experienceleague.adobe.com/docs/platform-learn/tutorials/data-ingestion/create-datasets-and-ingest-data.html?lang=en), and [segments](https://experienceleague.adobe.com/docs/platform-learn/tutorials/segments/create-segments.html?lang=en) created in [!DNL Experience Platform].
+
+Refer to Adobe's documentation for [Segment Membership Details schema field group](/help/xdm/field-groups/profile/segmentation.md) if you need guidance on segment statuses.
+
+### Outreach Prerequisites {#prerequisites-destination}
```
Comment (vfalon) [inline help/destinations/catalog/crm/outreach.md]: Sentence case for headings.

**Review by vfalon — CHANGES_REQUESTED** [review]
Please see my comments, as some of the previous ones seem to have been missed.

File: help/destinations/catalog/crm/outreach.md [hunk 31]
```diff
@@ -0,0 +1,219 @@
...
+1. Select the destination and validate that the status is **[!UICONTROL enabled]**.
+![Platform UI screenshot showing Destinations Dataflow Run for the selected destination.](../../assets/catalog/crm/outreach/destination-dataflow-run.png)
+
+1. Switch to the **[!DNL Activation data]** tab, then select a segment name.
+![Platform UI screenshot showing Destinations Activation data.](../../assets/catalog/crm/outreach/destinations-activation-data.png)
+
+1. Monitor the segment summary and ensure that the count of profiles corresponds to the count created within the segment.
+![Platform UI screenshot showing Segment summary.](../../assets/catalog/crm/outreach/segment.png)
+
+1. Log in to the [!DNL Outreach] website, then navigate to the **[!DNL Apps]** > **[!DNL Contacts]** page and check if the profiles from the segment have been added. You will also notice the segment status from Experience Platform has been updated against the corresponding custom field attribute *(identified through the friendly label name)* that was provided in the Platform **Mapping ID** field during the **[!UICONTROL Activate destination]** > **[!UICONTROL Schedule segment export]** step.
```
Comment (vfalon) [inline help/destinations/catalog/crm/outreach.md]: Line 196: I think I already mentioned this in my previous comments. There is no [!UICONTROL Activate destination] UI element.

Please rephrase the section as below:

"Log in to the [!DNL Outreach] website, then navigate to the **[!DNL Apps]** > **[!DNL Contacts]** page and check if the profiles from the segment have been added. You can see that each segment status in [!DNL Outreach] was updated with the corresponding segment status from Platform, based on the [!UICONTROL Mapping ID] value provided during the segment scheduling step."
